Introduction.md

```diff
@@ -0,0 +1,90 @@
+
+### Trust-minimized execution
+
+A second characteristic of DLCs is that both parties involved in the contract do not need to trust each other.
+The setup and execution of the contract is such that they are guaranteed to always be able to unilaterally close the contract.
+In case of a misbehaving counter-party, trying to close a contract on an outcome for which an oracle signature is not known, the honest protocol participant can claim all the funds locked in the contract.
+Parties are also protected against faulty oracles that would not produce a signature at event occurrence through a refund mechanism
+
+While able to unilaterally close the contracts, parties are expected to generally close the contract cooperatively, as it requires one less transaction to be broadcast.
+
+The current version of the specification does not offer protection against a party colluding with an oracle.
+Mitigation such as using a federation of oracles might be considered in future versions.
+
+Note that oracle cannot equivocate (create two different signatures on two different outcomes of a single event) without revealing their private key.
+This makes it possible to create a staking mechanism for oracles to penalize such behavior.
+
+## Glossary
+
+### Closing Transaction
+
+A transaction spending from the output of a [CET](#contract-execution-transaction-cet) requiring an oracle signature to be unlocked.
+This transaction thus unlocks the funds using the appropriate [oracle](#oracle) signature and sending them to an address fully controlled by the broadcasting party.
+It should usually be broadcast together with the corresponding CET.
+
+### Contract Execution Transaction (CET)
+
+A transaction spending from the [fund transaction](#fund-transaction) whose outputs represent a possible outcome of a DLC contract.
+The first output can be spent either by one of the contracting party with knowledge of an oracle signature over the outcome, or by their counter-party after a given timeout.
+The second output can be spent directly.
+
+### Funding transaction
+
+An irreversible on-chain transaction locking the contract collateral in an output that can only be spent by mutual consent.
+It can also contain change outputs in the case where the amount input by a party is greater than the desired collateral.
+
+### Mutual Close
+
+A cooperative closing of a DLC using a [mutual closing transaction](#mutual-closing-transaction).
+A mutual close does not require the broadcast of any [CET](#contract-execution-transaction-cet).
+
+### Mutual Closing Transaction
+
+A transaction spending from the [fund transaction](#fund-transaction) paying to each party the amount that was agreed based on the published event outcome.
+
+### Oracle
+
+Entity providing information to setup and decide the outcome of a DLC.
+Ahead of an event, an oracle publishes an [R-value](#r-value) that potential contracting parties can use to establish a DLC.
```

Comment:
Oracle should also publish A = aG also ? I mean public-key isn't only R-value. But going further would get down in specification itself I guess.

Reply:
Added something about public keys